xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -142,8 +190,8 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.Security.Cryptography.X509Certificates.X509Extension&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the X.509 Certificate Extensions to include in the certificate or certificate request.</summary>
```

Comment:
While this is a Get-only property, it's a mutable collection.  Callers are expected to add things to this collection.

I don't know what, if anything, that would mean changing here... but "Gets" threw me off, since it's really "gets the container into which they should be put" as opposed to "gets the things".